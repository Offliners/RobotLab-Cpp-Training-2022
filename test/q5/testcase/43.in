11-(13+9+6)/(14+18*15/16*6)=/(19*11)=-19+13*10-16*6/19-(2+10*14/9-(20-3)*9/8+1)*10*19*8/19+18/15+10+10*20%5=+19/12/(8*20)/13/17+15/(18-6+13)=*(6/16)+(9*12)%13=
