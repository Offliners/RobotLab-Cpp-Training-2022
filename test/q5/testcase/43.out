40
29
-31532
-63119
